src/main/webapp/app/file-upload-submission/file-upload-submission.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { JhiLanguageHelper } from 'app/core';
+import { ArtemisSharedModule } from 'app/shared';
+import { ArtemisResultModule } from 'app/entities/result';
+import { fileUploadSubmissionRoute } from './file-upload-submission.route';
+import { FileUploadSubmissionComponent } from './file-upload-submission.component';
+import { ArtemisComplaintsModule } from 'app/complaints';
+import { JhiLanguageService } from 'ng-jhipster';
+import { MomentModule } from 'ngx-moment';
+
+const ENTITY_STATES = [...fileUploadSubmissionRoute];
+
+@NgModule({
+    declarations: [FileUploadSubmissionComponent],
+    imports: [ArtemisSharedModule, RouterModule.forChild(ENTITY_STATES), ArtemisResultModule, ArtemisComplaintsModule, MomentModule],
+    providers: [{ provide: JhiLanguageService, useClass: JhiLanguageService }],
+})
+export class ArtemisFileUploadSubmissionModule {
+    constructor(private languageService: JhiLanguageService, private languageHelper: JhiLanguageHelper) {
```

Comment:
Please remove this. In the last `develop` update, we have removed this from all angular modules

Reply:
Removed